app/lib/submission_builder/ty2020/adv_ctc_irs1040.rb

```diff
@@ -1,5 +1,5 @@
 module SubmissionBuilder
-  module TY2020
+  module Ty2020
```

Comment:
when i changed from ty_2020 to ty2020, it started not liking the all upper cased casing on the TY2020 module name. Not sure if that's a reason to keep it the other way...?